hub/integrations/pytorch.py

```diff
@@ -186,11 +188,18 @@ def _np_from_chunk_list(self, index: int, key: str, chunks: List[bytes]):
         shape = index_entry["shape"]
 
         combined_bytes = join_chunks(chunks, start_byte, end_byte)
+
+        def _decompress_combined_bytes(combined_bytes: Union[bytes, memoryview]):
+            compressor = get_compressor(index_entry["compression"])
```

Comment:
isn't get_compressor called somewhere else?

Reply:
Yes, but the compressor object isn't stored anyway, only the compressor name is available here